gulpfile.js

```diff
@@ -766,6 +772,7 @@ gulp.task('chromium-pre', ['buildnumber', 'locale'], function () {
     preprocessHTML('web/viewer.html', defines)
         .pipe(gulp.dest(CHROME_BUILD_CONTENT_DIR + 'web')),
     preprocessCSS('web/viewer.css', 'chrome', defines, true)
+        .pipe(postcss([autoprefixer()]))
```

Comment:
This version only needs to be compatible with Chrome, so use:

```javascript
autoprefixer({ browsers: ['chrome >= 49'], })
```

(Chrome 49, because it is the last supported Chrome version by WIndows XP).